Decisions.Docusign/SendDocumentsForSigning.cs

```diff
@@ -195,9 +195,27 @@ public ResultData Run(StepStartData data)
                                     DocumentID = GetIdentifierOrDefaultOneAsString(ast.DocumentId),
                                 });
                             }
-                        };
+                        }
+                        
+                        // Add CC Recipients for current signer
+                        if (rtm.CCRecipients != null)
+                        {
+                            foreach (var ccRecipient in rtm.CCRecipients)
+                            {
+                                dsRecipients.Add(new Recipient
+                                {
+                                    Email = ccRecipient,
+                                    UserName = ccRecipient,
+                                    Type = RecipientTypeCode.CarbonCopy,
+                                    ID = (++recipientIndex).ToString(),
```

Comment:
why did you increment the recipient index here? it should be the same as recipient index

Reply:
Got it